src/binary/ibc-setup/commands/init.ts

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import path from 'path';
+
+import { Command } from 'commander';
+
+import { GlobalOptions } from '../types';
+
+type Options = GlobalOptions & {
+  src: string;
+  dest: string;
+};
+
+type ProgramOptions = Partial<Options>;
+
+function resolveHome(home?: string) {
+  if (home) {
+    return home;
+  }
+
+  if (!process.env.HOME) {
+    throw new Error('$HOME environment variable is not set.');
+  }
+
+  return `${process.env.HOME}/.ibc-setup`;
+}
+
+const REGISTRY_FILE = 'registry.yaml';
+
+export function init(_: unknown, program: Command) {
+  const programOptions: ProgramOptions = program.opts();
+
+  // TODO: Resolve options with precedence (env, command line etc.)
+  const options = {
+    src: programOptions.src ?? 'src',
+    dest: programOptions.dest ?? 'dest',
+    home: resolveHome(programOptions.home),
+  };
```

Comment:
The `ibc-setup init` will create`app.yaml` if it doesn't exist. It means, that we should use `app.yaml` while resolving options for this specific command. Is that right?

Reply:
@ethanfrey I updated the comment since it didn't make sense before.

Reply:
Yes, this is correct. You can use flags or env variables here.

You could just use the [same code I mention below](https://github.com/confio/ts-relayer/pull/73#discussion_r580853285), but replace with `const appConfig = {}` (eg. missing file)